src/Playroom/CodeEditor/CodeEditor.tsx

```diff
@@ -71,7 +71,7 @@ const validateCode = (editorInstance: Editor, code: string) => {
       marker.setAttribute(
         'title',
         // Remove our wrapping Fragment from error message
-        (err.message || '')
+        errorMessage
           .replace(/\<React\.Fragment\>/, '')
           .replace(/\<\/React\.Fragment\>$/, '')
```

Comment:
I think something might've broken here, seeing `</React.Fragment>` in error messages.
![image](https://user-images.githubusercontent.com/5663042/207179681-c82a712a-cf73-491c-b704-89a0ee420dfa.png)

Reply:
It appears to be in master too. I'll look into it